src/codegen/ruby/def/field.rb

```diff
@@ -41,6 +42,7 @@ def write(f)
     f.puts "  /**"
     f.puts "    * #{description}"
     f.puts "    */"
+    f.puts "  @javax.annotation.Nullable" if nullable?
```

Comment:
Do we really want to use javax.annotation or should we use the reactivex annotation?

Reply:
what you said is about it. Immutables depends on jsr305 to make use of the annotations provided there (which is where javax.annotation.Nullable comes from)

Reply:
Alrighty, don't see anything else I have concerns about.